### ISSUE TITLE
debian package missing dependency on libssl1.1

### ISSUE THREAD

**Issue by flub** [zerotier/ZeroTierOne#1802]
Installing the `zerotier-one` version `1.10.2` on debian bookworm using this apt line: `deb http://download.zerotier.com/debian/bookworm bookworm main` currently fails.  It is missing a dependency on the `libssl1.1` package and thus the binary is missing shared libraries:

```
/usr/sbin/zerotier-one: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```

Installing the `libssl1.1` package fixes this.

See also this comment: https://github.com/zerotier/ZeroTierOne/issues/1678#issuecomment-1242765506 but created as a separate issue since the original issue is about mint.

**Comment by LaurentBonnaud** [comment]
The `libssl1.1` package does not exist in Debian bookworm.  It has been replaced by `libssl3`.

zerotier should be rebuilt on a recent Debian system and the `/usr/sbin/zerotier-one` binary should be linked with the `libssl.so.3` library.

**Comment by ieugen** [comment]
I also have this issue.
Had it for some time now and I kept the libssl1.1 package over the upgrade.
Once I removed it zerotier broke. 
Now I don't have the service working on my machine :( .

I believe the builds for Bookworm do not actually build for Bookworm but for an older version of Debian.

**Comment by OfficialBoyfriend** [comment]
For elementary 7 (Ubuntu 22.04.1 LTS) users who come here through search engines, you can download and install libssl1.1 to solve the problem temporarily.

```shell
# download install package
wget http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1-1ubuntu2.1~18.04.20_amd64.deb
# install libssl1.1 deb package
sudo dpkg -i libssl1.1_1.1.1-1ubuntu2.1~18.04.20_amd64.deb
# install zerotier-one
curl -s https://install.zerotier.com | sudo bash
```

**Comment by flub** [comment]
To build your own debian package of zerotier: make a clone of the repository and invoke `make debian`  Doing so on bookworm  will create a debian package which works without any other tweaks.

You will need to install dependencies and debian tools etc of course.

**Comment by flub** [comment]
This is fixed in version 1.10.3

**Comment by m4lvin** [comment]
In which commit was this fixed? I have zerotier-one 1.10.6 and have this problem, after upgrading Debian from bullseye to bookworm and removing the now unavailable package libssl1.1.
